lib/liquider/tags/cycle_tag.rb

```diff
@@ -0,0 +1,13 @@
+class CycleTag < Liquider::Tag
+  def render_erb(compiler)
+    compiler.erb_tag(output: true) do
+      compiler.raw(
+        "(@cycles ||= Hash.new{|h,k|h[k] = [0, k]}; k = ["
+      )
+      compiler.on_arg_list(markup)
```

Comment:
This makes me want to refactor the compiler to be functional even more. Especially since compile time is not so important